tests/with_ray/on_wlm/test_ray_pbs.py

```diff
@@ -55,7 +55,7 @@ def test_ray_launch_and_shutdown(fileutils, wlmutils, caplog):
 
     exp.generate(cluster)
     exp.start(cluster, block=False, summary=False)
-    ctx = ray.client(cluster.get_head_address() + ":10001").connect()
+    ctx = ray.client("ray://" + cluster.get_head_address() + ":10001").connect()
```

Comment:
should we just be including this ``ray://`` bit in the ``cluster.get_head_address()`` function? port also?

Any reason not to?

Reply:
I think it is better to keep it like this, because if it is local, the port is different (and you don't add `ray://`).